bfv/example_parameters.go

```diff
@@ -0,0 +1,13 @@
+package bfv
+
+var (
+	// ExampleParameters128BitLogN14LogQP438 is an example parameters set with logN=14, logQP=438
```

Comment:
I would suggest re-adding different sets of secure example parameters. This was really practical in order to find out depending on the use case which parameters were the best fit

Reply:
More specifically it would be good to have 3-4 sets of parameters (as it was the case with the previous "default" params) but have user-friendly comments that explain the details of these parameters:
- number of slots
- estimate of the maximum depth (if applicable)
- if applicable, which parameters are flexible/can be changed without impacting the security

Reply:
I believe it would be better to have an example explaining how to parameterize the scheme.